tests/integration/security/jwt_test.go

```diff
@@ -46,17 +46,17 @@ const (
 func TestRequestAuthentication(t *testing.T) {
 	payload1 := strings.Split(jwt.TokenIssuer1, ".")[1]
 	payload2 := strings.Split(jwt.TokenIssuer2, ".")[1]
+
 	framework.NewTest(t).
 		Features("security.authentication.jwt").
 		Run(func(t framework.TestContext) {
-			ns := apps.Namespace1
+			ns := istio.ClaimSystemNamespaceOrFail(t, t)
```

Comment:
hmm, probably better to use a different variable for the istio-system namespace, otherwise you will also create the policies in the istio-system namespace which would work since it's the root namespace but I'm not sure this is needed for this test.

Reply:
so the same namespace is being used while iterating the testcases. That's why it was failing previously. If I add a new var for istio-system, I need to add a condition when we are iterating the testcases.

Reply:
why, you just create a new var for istio-system and use it for the jwt-server, the policy will still use the old ns, how could this fail? The namespace of the jwt-server should not affect the policy namespace.